views/program_index.ejs

```diff
@@ -24,6 +25,11 @@
                 <td><%= programs[i].end_date %></td>
                 <td><%= programs[i].min_grade_level %></td>
                 <td><%= programs[i].max_grade_level %></td>
+				<td>
+					<% for (let j = 0; j < programs[i].student_list.length; j++) { %>
+                        <a href="/view/<%= programs[i].student_list[j].substring(programs[i].student_list[j].indexOf("-") + 2) %>"><%= programs[i].student_list[j].substring(0,programs[i].student_list[j].indexOf("-") - 1) %></a>
```

Comment:
Similar to above, not exactly sure what this is doing but there might be a way to simplify it.